packages/firestore/android/src/main/java/io/invertase/firebase/firestore/UniversalFirebaseFirestoreStatics.java

```diff
@@ -22,4 +22,5 @@
   public static String FIRESTORE_HOST = "firebase_firestore_host";
   public static String FIRESTORE_PERSISTENCE = "firebase_firestore_persistence";
   public static String FIRESTORE_SSL = "firebase_firestore_ssl";
```

Comment:
this is unrelated to your PR but I just saw this is `ss1` in the string for `FIRESTORE_SSL` - it appears no one uses this, or they are just also using the typo string. Will peel this off for a separate issue

Reply:
Pretty sure it just says ssl

Reply:
Yes - looks clear to me here - I was looking at it on the phone and perhaps the font was off or my eyes are just going. Anyway, agreed - no issue